awx/ui_next/src/screens/Team/TeamList/TeamListItem.jsx

```diff
@@ -0,0 +1,87 @@
+import React, { Fragment } from 'react';
+import { string, bool, func } from 'prop-types';
+import { withI18n } from '@lingui/react';
+import { t } from '@lingui/macro';
+import {
+  DataListItem,
+  DataListItemRow,
+  DataListItemCells,
+  Tooltip,
+} from '@patternfly/react-core';
+import { Link } from 'react-router-dom';
+import { PencilAltIcon } from '@patternfly/react-icons';
+
+import ActionButtonCell from '@components/ActionButtonCell';
+import DataListCell from '@components/DataListCell';
+import DataListCheck from '@components/DataListCheck';
+import ListActionButton from '@components/ListActionButton';
+import VerticalSeparator from '@components/VerticalSeparator';
+import { Team } from '@types';
+
+class TeamListItem extends React.Component {
+  static propTypes = {
+    team: Team.isRequired,
+    detailUrl: string.isRequired,
+    isSelected: bool.isRequired,
+    onSelect: func.isRequired,
+  };
+
+  render() {
+    const { team, isSelected, onSelect, detailUrl, i18n } = this.props;
+    const labelId = `check-action-${team.id}`;
+    return (
+      <DataListItem key={team.id} aria-labelledby={labelId}>
+        <DataListItemRow>
+          <DataListCheck
+            id={`select-team-${team.id}`}
+            checked={isSelected}
+            onChange={onSelect}
+            aria-labelledby={labelId}
+          />
+          <DataListItemCells
+            dataListCells={[
+              <DataListCell key="divider">
+                <VerticalSeparator />
+                <Link
+                  id={labelId}
+                  to={`${detailUrl}`}
+                  style={{ marginRight: '10px' }}
```

Comment:
We have been mixing `style=` and `css=`. Personally, I prefer `style=` because it allows us to keep the CSS syntax.

Reply:
We actually don't even need this style.... will remove